android/guava/src/com/google/common/net/HttpHeaders.java

```diff
@@ -140,7 +140,7 @@ private HttpHeaders() {}
   public static final String PROXY_AUTHORIZATION = "Proxy-Authorization";
   /** The HTTP {@code Range} header field name. */
   public static final String RANGE = "Range";
-  /** The HTTP {@code Referer} header field name. */
+  /** The HTTP {@code Referrer} header field name. */
```

Comment:
https://en.wikipedia.org/wiki/HTTP_referer

Reply:
ok sir, i will fix it !